projects/demo/src/modules/services/alerts/method.md

```diff
@@ -1,6 +1,6 @@
 ```text
-show<O, I>(
-  content: PolymorpheusContent<TuiNotificationContentContext<O, I>>,
-  options: TuiNotificationOptions | TuiNotificationOptionsWithData<I>
-): Observable<O>;
+open<G = void>(
+    content: PolymorpheusContent<TuiBaseDialogContext<G> & T>,
```

Comment:
This looks like the abstract options. Let's put the actual alert interfaces here.

Reply:
fixed